opm/autodiff/BlackoilModelBase.hpp

```diff
@@ -294,16 +294,33 @@ namespace Opm {
         std::vector<V>
         variableStateInitials(const ReservoirState& x,
                               const WellState& xw) const;
+        void
+        variableReservoirStateInitials(const ReservoirState& x,
+                                       std::vector<V>& vars0) const;
+        void
+        variableWellStateInitials(const WellState& xw,
+                                  std::vector<V>& vars0) const;
 
         std::vector<int>
         variableStateIndices() const;
 
+        std::vector<int>
+        variableWellsStateIndices() const;
+
+        void
+        addWellContribution2MassBalanceEq(const std::vector<ADB>& cq_s);
```

Comment:
I know I have been guilty of it in the past, but I have come to the conclusion that using `2` to mean `To` is a bad idea. Please rename this (and we should make the same renaming in older code, too).